src/components/MainMenu/Menu.module.scss

```diff
@@ -2,22 +2,33 @@
 @import "../../font-size.scss";
 
 .list {
-  list-style: none;
-  font-size: $small;
+  list-style: none; // senza bullet points!!
+  font-size: $medium;
   color: $secondary;
   font-weight: bold;
+  padding-left: 0;
 }
 
 .elem {
-  padding: 5px 0px;
+  opacity: 0.5;
+  cursor: pointer;
+  padding-bottom: 20px;
 }
 
 .title {
   color: $secondary;
   font-size: $large;
   font-weight: bold;
+  margin-bottom: 60px;
 }
 
 .main {
   display: block;
 }
+
+.selected {
+  opacity: 1;
+  padding-left: 20px;
+  padding-bottom: 20px;
+  cursor: pointer;

```

Comment:
queste due regole in effetti sono duplicate. Puoi rimuoverle una volta sistemato il modo in cui applichi le regole ai menu items

Reply:
Grazie!!